src/transform/src/lib.rs

```diff
@@ -387,11 +387,6 @@ impl Default for FuseAndCollapse {
                 // Removes redundant inputs from joins.
                 // Note that this eliminates one redundant input per join,
                 // so it is necessary to run this section in a loop.
-                // TODO: (#6748) Predicate pushdown unlocks the ability to
```

Comment:
@petrosagg This comment came from https://github.com/MaterializeInc/materialize/commit/61dd8ef73311e06ced9e9294152225e2b8d3d88f, I tried removing the RedundantJoin below, but noticed that plans were showing redundant joins? See https://buildkite.com/materialize/tests/builds/67234#018b80bb-5ac2-403d-814d-08f85929a2b2